src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-net6/.vsconfig

```diff
@@ -1,68 +1,94 @@
 {
   "version": "1.0",
   "components": [
-    "android",
-    "android.aot",
-    "ios",
-    "maui.core",
-    "maui.android",
-    "maui.ios",
-    "maccatalyst",
-    "maui.maccatalyst",
-    "maui.windows",
-    "microsoft.net.runtime.android",
-    "microsoft.net.runtime.android.aot",
-    "microsoft.net.runtime.mono.tooling",
-    "microsoft.net.sdk.emscripten",
-    "microsoft.net.runtime.ios",
-    "microsoft.net.runtime.maccatalyst",
-    "runtimes.ios",
-    "runtimes.maccatalyst",
-    "wasm.tools",
-    "Component.OpenJDK",
-    "Component.Android.SDK.MAUI",
-    "Component.Xamarin.RemotedSimulator",
-    "Microsoft.Component.MSBuild",
-    "Microsoft.Component.NetFX.Native",
-    "Microsoft.ComponentGroup.Blend",
-    "Microsoft.NetCore.Component.DevelopmentTools",
-    "Microsoft.NetCore.Component.Runtime.5.0",
-    "Microsoft.NetCore.Component.Runtime.6.0",
+    "Microsoft.VisualStudio.Component.CoreEditor",
+    "Microsoft.VisualStudio.Workload.CoreEditor",
+    "Microsoft.NetCore.Component.Runtime.3.1",
```

Comment:
You can remove this change (+ other `.vsconfig` files) from your PR. Looks like an unwanted/unrelated change.

Reply:
They indeed are unrelated, but I don't know how to remove them from the PR...